core/src/main/scala/org/apache/spark/storage/BlockManager.scala

```diff
@@ -723,7 +728,9 @@ private[spark] class BlockManager(
       }
 
       if (data != null) {
-        return Some(new ChunkedByteBuffer(data))
+        val chunkSize =
+          conf.getSizeAsBytes("spark.storage.memoryMapLimitForTests", Int.MaxValue.toString).toInt
```

Comment:
nit: Make `chunkSize` as a `private` field in `BlockManager` instead of recomputing it each time ?